### DISCUSSION TITLE
Google Link for Coordinates

### DISCUSSION THREAD

**Discussion by RickAust** [deckerst/aves#1118] [Ideas]
Hello. I love Aves but I wish there was an easy way to incorporate GPS coordinates into Google Maps. I often add Favourites in Google Maps, based on the GPS coordinates of photos I've taken. I've tried manually copying and pasting the Degrees, minutes, and seconds (DMS) data into Google Maps but, not only is it time-consuming but I've found it wildly inaccurate (which is strange, as it looks accurate on the Aves map). Maybe, it's because it uses the DMS fotmat, rather than Decimal Degrees (DD). Not that it should make any difference. But maybe GM prefers the DD format.
Anyway, what I'd love is to be able to tap on the coordinates in the Aves site, which would instantly open open GM and allow me to save the site.
If this is problematic, the next best option would be to tap on the location in Decimal Degrees and instantly save onto my phone's clipboard.
Just a suggestion.

**Comment by deckerst** [comment]
Recently discussed in #1087.
Basically, there already was an action (via the viewer menu), and I've added a button for this on the map itself in the latest release v1.11.9.

**Comment by RickAust** [comment]
Sorry about that. I must have an old version.
That's great. Cheers 👍🙂

**Reply by deckerst** [reply]
v1.11.9 is not yet on the play store, I've just released it yesterday.

**Comment by RickAust** [comment]
OK. Thanks again 👍🙂
